src/components/Game/Game.tsx

```diff
@@ -55,62 +53,60 @@ export const Game = () => {
     dispatch(fetchDeckId());
   }, [dispatch]);
 
-  const onStartSubmit = () => {
+  const onStartNewGameSubmit = () => {
+    setGameStatus(GameStatus.start);
+    unsetPreviousGame();
+  };
+
+  const unsetPreviousGame = () => {
     setWinner([]);
-    // setCountDealer(0);
+    setCountDealer(0);
     setPlayerWin(false);
-    setGameStatus(GameStatus.start);
     dispatch(unsetUserHand());
     setCardsForPlayer([]);
   };
 
-  const onFirstSubmit = async () => {
+  const onBetFormSubmit = async () => {
     setGameStatus(GameStatus.inprogress);
     setCardsForPlayer(await cardsApi.getNewCard(deckId, 2));
+    setCountDealer(await setDealersHand());
   };
 
-  const onSubmit = async () => {
+  const onStaySubmit = async () => {
     setCardsForPlayer(await cardsApi.getNewCard(deckId, 1));
   };
 
-  const setResultForPlayer = () => {
-    winner?.forEach((player) => {
-      player.name === email ? setPlayerWin(true) : setPlayerWin(false);
-    });
-    // do not changes into true
-  };
-
-  const onStopSubmit = async () => {
+  const onHitSubmit = () => {
```

Comment:
Hit? What it means?

Reply:
i was searching for game examples to try to play, and in each game i saw two buttons "hit" and "stay". So to understand which function will work on which button submit - i decided to name functions as buttons